packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js

```diff
@@ -110,12 +111,12 @@ class DateTimeView extends React.PureComponent {
 			rightElement: (
 				<Action
 					aria-label={t('DATEPICKER_MONTH_NEXT', { defaultValue: 'Go to next month' })}
-					icon="talend-chevron-left"
+					icon="talend-arrow-left"
 					iconTransform="rotate-180"
 					label=""
 					onClick={() => this.goToNextMonth()}
 					tabIndex="-1"
-					link
+					className="btn-tertiary btn-info"
```

Comment:
The right arrow is not described as a tertiary button in guideline

Reply:
Hi, The left/right arrow is tertiary button in guideline.